qa/tasks/cephfs/test_client_limits.py

```diff
@@ -221,6 +222,49 @@ def test_client_release_bug(self):
         self.fs.mds_asok(['session', 'evict', "%s" % mount_a_client_id])
         rproc.wait()
 
+    def test_client_blocklisted_oldest_tid(self):
+        """
+        that a client is blocklisted when its encoded session metadata exceeds the
+        configured threshold (due to ever growing `completed_requests` caused due
+        to an unidentified bug (in the client or the MDS)).
+        """
+
+        # num of requests client issues
+        max_requests = 10000
+
+        # The debug hook to inject the failure only exists in the fuse client
+        if not isinstance(self.mount_a, FuseMount):
+            self.skipTest("Require FUSE client to inject client release failure")
+
+        self.set_conf('client', 'client inject fixed oldest tid', 'true')
```

Comment:
I'm trying to encourage avoiding `set_conf` whenever possible since it changes the `ceph.conf` on the hosts. From my glance at the code, I think

```suggestion
        self.config_set('client', 'client inject fixed oldest tid', 'true')
```

Should work?

Reply:
Fair enough. I mostly duplicated this tests from the one just after it.